# The custom msg file created by isy on 12/16/2017
# it gives the desired velocitie to the controller
# desired_velocity is the linear velocity (x,y,z)
Header header
geometry_msgs/Point pdes
geometry_msgs/Point vdes
geometry_msgs/Point ades
geometry_msgs/Point jdes
geometry_msgs/Point sdes
geometry_msgs/Point ddes
geometry_msgs/Point ddes_dot
int32 controller
